iroh-ctl/Cargo.toml

```diff
@@ -6,14 +6,14 @@ authors = ["Kasey Huizinga <klhuizinga@gmail.com>", "Martijn Faassen <faassen@st
 license = "Apache-2.0/MIT"
 repository = "https://github.com/n0-computer/iroh"
 description = "Client for interacting with running iroh processes."
-default-run = "iroh-ctl"
+default-run = "iroh"
 
 [features]
 testing = []
 
 [[bin]]
 path = "src/main.rs"
-name = "iroh-ctl"
```

Comment:
lets skip this in this PR, or do the full renaming please

Reply:
if I promise I will do the full renaming can I keep the name = "iroh" (for the binary) in there for now? Otherwise I have to move my trycmd test into the wrong direction back to iroh-ctl, and I already wrote a few more on followup work

Reply:
Example of what I'll need to change back:

```
$ iroh get /ipfs/QmP8jTG1m9GSDJLCbeWhVSVgEzCPPwXRdCRuJtQ5Tz9Kc9
Saving file(s) to QmP8jTG1m9GSDJLCbeWhVSVgEzCPPwXRdCRuJtQ5Tz9Kc9

```

Note that trycmd has an extremely annoying behavior of silently skipping a trycmd test if the binary is missing. :(

Reply:
(discussed on chat, it's okay for now)